src/test/resources/features/ValidateShelterCertificate.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Validate ShelterCertificate
+  As an Admin
+  I want to validate a ShelterCertificate
+  So that I can ensure Its validity and authenticity
+
+  Background: Exists a shelter with name "test"
+    Given A shelter with name "test"
+    And An admin with username "admin" and password "123456789"
+
+  Scenario: Validating a correct shelter certificate
+    Given a shelter certificate associated with a shelter with name "test" with valid information created by a shelter volunteer
+    When the admin with username "admin" and password "123456789" verifies the certificate validity associated with a shelter with name "test"
+    Then The response code is 200
+
+  Scenario: Validating a incorrect shelter certificate
+    Given a shelter certificate associated with a shelter with name "test" with invalid information created by a shelter volunteer
+    Then the admin with username "admin" and password "123456789" rejects the certificate validity associated with a shelter with name "test"
```

Comment:
Shouldn't this be a When and not a Then? And, accordingly, in the implementation of this step use the API to make some sort of update of the state of the certificate (beyond just reading it)?

Reply:
Solved the feature definition and improved the step implementation.